packages/flutter/test/material/outline_button_test.dart

```diff
@@ -0,0 +1,252 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
Could we combine some of the standard tests for all types of buttons we have? I imagine basic things like sizing, gesture recognition, semantics and ink splash would behave the same.

Reply:
Both the button implementations and their tests could be factored better. That would be a separate PR.